### PR TITLE
chore: release v1.6.0

### PR DESCRIPTION
**Pull request by pb8o**
## Changes

Bookkeeping changes to make a release.

See `CHANGELOG.md` for the actual list of changes included in the release..

## Reason

Release latest features and bug fixes.

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license. For more information on following
Developer Certificate of Origin and signing off your commits, please check
[`CONTRIBUTING.md`][3].

## PR Checklist

- [ ] If a specific issue led to this PR, this PR closes the issue.
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.
- [x] API changes follow the [Runbook for Firecracker API changes][2].
- [x] User-facing changes are mentioned in `CHANGELOG.md`.
- [x] All added/changed functionality is tested.
- [x] New `TODO`s link to an issue.
- [x] Commits meet [contribution quality standards](https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md#contribution-quality-standards).

---

- [x] This functionality cannot be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md
[3]: https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: CREDITS.md [hunk 1]
```diff
@@ -17,9 +17,11 @@ written in Rust with a focus on safety and security. Thanks go to:
 
 Contributors to the Firecracker repository:
 
+* Aaqa Ishtyaq <aaqaishtyaq@gmail.com>
 * Aaron Hill <aa1ronham@gmail.com>
 * Aaron O'Mullan <aaron.omullan@gmail.com>
 * Abhijeet Kasurde <akasurde@redhat.com>
+* acatangiu <adrian@parity.io>
```
Comment (pb8o) [inline CREDITS.md:24]: Should we merge this entry with the below? Which email should be the preferred one?

Comment (roypat) [reply CREDITS.md:24]: I'd vote merge and use new email

Comment (bchalios) [reply CREDITS.md:24]: I think we should maybe leave both there, so as to indicate that different work was done under the umbrella of different companies?

Comment (pb8o) [reply CREDITS.md:24]: I will leave it in this release for now, and we can think of fixing it in main if need be.

**Review by roypat — APPROVED** [review]

**Review by sudanl0 — APPROVED** [review]

**Pull request merged**